scene/3d/navigation.cpp

```diff
@@ -340,16 +340,12 @@ Vector<Vector3> Navigation::get_simple_path(const Vector3 &p_start, const Vector
 			};
 
 			Vector3 entry = Geometry::get_closest_point_to_segment(begin_poly->entry, edge);
-			begin_poly->edges[i].C->distance = begin_poly->entry.distance_to(entry);
+			begin_poly->edges[i].C->distance = being_point.distance_to(entry);
```

Comment:
Typo, fails compiling.

Reply:
huh, I'm pretty sure I tested the last version as well....